pkg/migrate/ListFiles.go

```
@@ -0,0 +1,48 @@
+package migrate
+
+import (
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+// ListFiles lists the files in a given directory.
+func ListFiles(p string, s3Client *s3.S3) ([]string, error) {
+	if strings.HasPrefix(p, "file://") {
+		f, err := os.Open(p[len("file://"):])
+		if err != nil {
+			return []string{}, err
+		}
+		return f.Readdirnames(-1)
+	} else if strings.HasPrefix(p, "s3://") {
+		parts := strings.SplitN(p[len("s3://"):], "/", 2)
+		if len(parts) == 2 {
```

Should we throw an error if it doesn't equal 2? However, I think that its likely impossible so do we want to just skip this `if` statement?